Weather Alert/CoreData/DataController.swift

```diff
@@ -80,6 +80,33 @@ class DataController {
         }
         return true
     }
+}
+
+extension DataController {
     
+    var dataController: DataController { return AppShared.instances.dataController }
```

Comment:
I seem to recall we changed this to be a class variable.
Accessing it from outside will require you to create a new instance every time just to access the same static variable stored in app shared. 
Don't know why it's not like this anymore. It should be.